starter-code/bin/seed.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require('mongoose')
+
+// const celebrityModel = require('../models/celebrity')
+
+const movieModel = require('../models/movie')
```

Comment:
as best practices, models should always start with uppercase (pascal casing)
`const MovieModel = require('../models/movie')`